cmd/tracee/cmd/list.go

```diff
@@ -0,0 +1,22 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	tcmd "github.com/aquasecurity/tracee/pkg/cmd"
+)
+
+func init() {
+	rootCmd.AddCommand(listCmd)
+}
+
+var listCmd = &cobra.Command{
+	Use:     "list",
+	Aliases: []string{"l"},
+	Short:   "List traceable events",
+	Long:    ``,
+	Run: func(cmd *cobra.Command, args []string) {
+		tcmd.PrintEventList(true) // list events
```

Comment:
This is a problem for signature events. They need to be loaded before listing or else we will always see an empty list. 

There was an idea to make the default signature events part of tracee instead of a plugin, which would fix this for tracee signature events, for users' events we still need to ask a path and load the events before.

Reply:
Thanks @josedonizetti. fixed (workaround) via c55bc45d71522ee267d86cb55fde3848a67bb87d.